Update release cheat sheet (#7425)
tektoncd/pipeline: tekton/release-cheat-sheet.md

1. `cd` to root of Pipelines git checkout.

> 1. [Install kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/source/#install-the-kustomize-cli-from-local-source-with-cloning-the-repo) and use it to ensure the correct version of the release pipeline is installed on the cluster:

Reviewer: ```suggestion
1. [Install kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/source/#install-the-kustomize-cli-from-local-source-with-cloning-the-repo) if you haven't already.

1. Ensure the correct version of the release pipeline is installed on the cluster:
```

1. `cd` to root of Pipelines git checkout.

> 1. [Install kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/source/#install-the-kustomize-cli-from-local-source-with-cloning-the-repo) if you haven't already.

Reviewer: let them choose whichever way they want to install it 

```suggestion
1. [Install kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) if you haven't already.
```
